install4j-maven-plugin/src/main/java/org/sonatype/install4j/maven/CompileMojo.java

```diff
@@ -286,6 +291,28 @@ protected void execute(final AntHelper ant, final ExecTask task) throws Exceptio
     }
   }
 
+  private void createVariablesTempFile() throws IOException {
+    variablesTempFile = File.createTempFile("install4jVariables", ".txt");
+    FileOutputStream out = null;
+    try {
+      out = new FileOutputStream(variablesTempFile);
+      out.write(BOM); //BOM forces UTF-8 detection for reading
```

Comment:
Is this a nice to have or is install4j needing this for a reason?

https://en.wikipedia.org/wiki/Byte_order_mark#cite_note-3 suggests BOM for UTF-8 is not needed or recommended.

I'm fine to leave this in if you believe its needed, though I've never had to do such for any UTF-8 files before.